proto/store/idp.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package bytebase.store;
+option go_package = "generated-go/store";
+
+// FieldMapping uses for mapping user info from identity provider to Bytebase.
+message FieldMapping {
```

Comment:
Is this field mapping generic enough?

Identity is the field from auth provider. What does email or name mean?

Reply:
`display_name` and `email` are common response data in OAuth2's user information requests. But for different providers, there are different fields in the response body. So we need to them in the mapping structure.